pixleesdk/src/main/java/com/pixlee/pixleesdk/data/api/AnalyticsAPI.java

```diff
@@ -0,0 +1,24 @@
+package com.pixlee.pixleesdk.data.api;
```

Comment:
With all of these new files, can we add some top level comments explaining the file and what it does?

Reply:
I added this class to write mock unit tests in the Unit Test area for the next update. In unit tests usually we should avoid using Android SDK related codes unless Android APIs support testing.